### PR TITLE
[zh-cn] sync access-authn-authz/validating-admission-policy.md

### PR DESCRIPTION
**Pull request by my-git9**

### CODE REVIEW

**Comment by tengqm** [comment]
These look like some new examples referenced by a new/updated page. Please sync that page(s) together with these examples.

**Comment by my-git9** [comment]
> These look like some new examples referenced by a new/updated page. Please sync that page(s) together with these examples.

Thanks for review, I will do it later

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 1]
```diff
@@ -446,6 +403,64 @@ Here, we first check that the optional parameter is present with `!has(params.op
...
+In addition to specify a parameter in a binding by `name`, you may
+choose instead to specify label selector, such that all resources of the
+policy's `paramKind`, and the param's `namespace` (if applicable) that match the
+label selector are selected for evaluation. See {{< glossary_tooltip text="selector" term_id="selector">}} for more information on how label selectors match resources.
+-->
+#### 参数选择器
+
+除了在绑定中指定 `name` 参数之外，你还可以选择指定标签选择器，
+这样策略的 `paramKind` 参数所指定的所有资源以及与标签选择器匹配的参数的
+`namespace`（如果适用）参数进行评估。有关标签选择器如何匹配资源的更多信息，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:447]:
```suggestion
这样对于与策略的 `paramKind` 参数匹配，且位于参数的 `namespace`（如果适用）内的所有资源，
如果与标签选择算符匹配，都会被评估。有关标签选择算符如何匹配资源的更多信息，
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 2]
```diff
@@ -446,6 +403,64 @@ Here, we first check that the optional parameter is present with `!has(params.op
...
+请参阅 {{<glossary_tooltip text="selector" term_id="selector">}}。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:448]:
```suggestion
请参阅{{<glossary_tooltip text="选择算符" term_id="selector">}}。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 3]
```diff
@@ -446,6 +403,64 @@ Here, we first check that the optional parameter is present with `!has(params.op
...
+<!--
+If multiple parameters are found to meet the condition, the policy's rules are
+evaluated for each parameter found and the results will be ANDed together.
+
+If `namespace` is provided, only objects of the `paramKind` in the provided
+namespace are eligible for selection. Otherwise, when `namespace` is empty and 
+`paramKind` is namespace-scoped, the `namespace` used in the request being 
+admitted will be used.
+-->
+如果发现多个参数满足条件，则会针对找到的每个参数评估策略规则，并将结果进行“与”运算。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:459]:
```suggestion
如果发现多个参数满足条件，则会针对所找到的每个参数来评估策略规则，并将结果进行“与”运算。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 4]
```diff
@@ -446,6 +403,64 @@ Here, we first check that the optional parameter is present with `!has(params.op
...
+
+如果设置了 `namespace`，则只有所提供的命名空间中 `paramKind` 指定的对象才会被匹配。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:461]:
```suggestion
如果设置了 `namespace`，则只有所提供的命名空间中类别为 `paramKind` 的对象才会被匹配。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 5]
```diff
@@ -446,6 +403,64 @@ Here, we first check that the optional parameter is present with `!has(params.op
...
+否则，当 `namespace` 为空且 `paramKind` 为命名空间范围时，将使用正在接受的请求中指定的 `namespace`。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:462]:
```suggestion
否则，当 `namespace` 为空且 `paramKind` 为命名空间作用域的资源时，使用被准入请求中指定的 `namespace`。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 6]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
   对 CRD 的支持将在未来发布中推出。
+
+<!--
+### Variable composition
+
+If an expression grows too complicated, or part of the expression is reusable and computationally expensive to evaluate,
+you can extract some part of the expressions into variables. A variable is a named expression that can be referred later
+in `variables` in other expressions.
+-->
+### Variable 组合
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:903]:
```suggestion
### 变量组合
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 7]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+
+如果表达式变得太复杂，或者表达式的一部分可重用且计算成本高昂，则可以将表达式的某些部分提取到 variables 中。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:905]:
```suggestion
如果表达式变得太复杂，或者表达式的一部分可重用且进行评估时计算开销较大，可以将表达式的某些部分提取为变量。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 8]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+variable 是一个命名表达式，稍后可以在其他表达式中的 `variables` 中引用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:906]:
```suggestion
变量是一个命名表达式，后期可以在其他表达式中的 `variables` 中引用。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 9]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+A variable is lazily evaluated when it is first referred. Any error that occurs during the evaluation will be
+reported during the evaluation of the referring expression. Both the result and potential error are memorized and
+count only once towards the runtime cost.
+
+The order of variables are important because a variable can refer to other variables that are defined before it.
+This ordering prevents circular references.
+
+The following is a more complex example of enforcing that image repo names match the environment defined in its namespace.
+-->
+variable 在首次引用时会被延迟求值。评估期间发生的任何错误都将在评估引用表达式期间报告，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:927]:
```suggestion
变量在首次引用时会被延迟求值。评估期间发生的任何错误都将在评估引用表达式期间报告，
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 10]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+其结果和潜在错误都会被记录，并且在运行时成本中仅计入一次。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:928]:
```suggestion
结果和可能的错误都会被记录下来，且在运行时开销中仅计为一次。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 11]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+
+variable 的顺序很重要，因为一个 variable 可以引用在它之前定义的其他 variable。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:930]:
```suggestion
变量的顺序很重要，因为一个变量可以引用在它之前定义的其他变量。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 12]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+这种顺序可以防止循环引用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:931]:
```suggestion
对顺序的要求可以防止循环引用。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 13]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+
+以下是强制镜像仓库名称与其命名空间中定义的环境相匹配的更复杂示例。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:933]:
```suggestion
以下是强制镜像仓库名称与其命名空间中定义的环境相匹配的一个较复杂示例。
```

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 14]
```diff
@@ -923,3 +892,64 @@ Type Checking has the following limitation:
...
+
+{{< codenew file="access/image-matches-namespace-environment.policy.yaml" >}}
+
+<!--
+With the policy bound to the namespace `default`, which is labeled `environment: prod`,
+the following attempt to create a deployment would be rejected.
+-->
+通过将策略绑定到 `default` 命名空间（标记为 `environment: prod`），
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:941]:
```suggestion
在此策略被绑定到 `default` 命名空间（标签为 `environment: prod`）的情况下，
```

**Comment by tengqm** [comment]
Looks like there are still 13 conversations not resolved?

**Comment by my-git9** [comment]
> Looks like there are still 13 conversations not resolved?

All conversations has been resolved

**Comment by tengqm** [comment]
/lgtm
Please trigger an empty commit to rerun the gate tests.

**Review by ydFu — APPROVED** [review]
/approve

**Comment by ydFu** [comment]
> /lgtm Please trigger an empty commit to rerun the gate tests.

@my-git9 
Technical review has been completed, please follow @tengqm advice.

**Pull request merged**